### PR TITLE
Bump to Pulsar 2.7.4

### PR DESCRIPTION
**Pull request by Technoboy-**
Update the Helm Chart according to the release process
https://github.com/apache/pulsar/wiki/Release-process#11-release-helm-chart

### CODE REVIEW

File: charts/pulsar/Chart.yaml [hunk 1]
```diff
@@ -18,10 +18,10 @@
 #
 
 apiVersion: v1
-appVersion: "2.7.2"
+appVersion: "2.7.4"
 description: Apache Pulsar Helm chart for Kubernetes
 name: pulsar
-version: 2.7.6
+version: 2.7.4
```
Comment (eolivelli) [inline charts/pulsar/Chart.yaml:24]: you cannot set a value minor then 2.7.6

**Review by eolivelli — CHANGES_REQUESTED** [review]

**Review by eolivelli — APPROVED** [review]
+1

File: charts/pulsar/Chart.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@
 #
 
 apiVersion: v1
-appVersion: "2.7.2"
+appVersion: "2.7.4"
 description: Apache Pulsar Helm chart for Kubernetes
 name: pulsar
 version: 2.7.6
```
Comment (lhotari) [inline charts/pulsar/Chart.yaml:24]: @Technoboy- you need to bump the chart version to 2.7.7

**Review by lhotari — CHANGES_REQUESTED** [review]

File: charts/pulsar/Chart.yaml (hunk 2 above)
Comment (Technoboy-) [reply charts/pulsar/Chart.yaml:24]: Did you mean :  version: 2.7.6 -> version: 2.7.7?

Comment (Technoboy-) [reply charts/pulsar/Chart.yaml]: Ok, thanks @lhotari

**Review by lhotari — APPROVED** [review]

**Pull request merged**